### DISCUSSION TITLE
Special scene questions about minio

### DISCUSSION THREAD

**Discussion by Hello-Linux** [minio/minio#19483] [Q&A] [answered]
We have a local self-developed file system with two hosts each having a mount point /data1 of the file system. minio uses the mount points on these two nodes to build a cluster. Now, my problem is that our file system regularly performs truncate operation on files in the mount point directory. After this operation, the number of bytes of the file is 0, but the metadata and file name of all files such as inode are present. In this case, can the client initiate a get request to retrieve the file?

**Comment by klauspost** [comment, marked as answer]
No idea what you are talking about here. If you do your own FS you are on your own.

**Reply by Hello-Linux** [reply]
**ok, the name is file stub**
A reserved file stub is a feature that allows a file to have a placeholder in the file system, but the actual data is stored elsewhere, such as in a tape archive or cloud storage. When you truncate a file, the reserved file stub retains the placeholder for the file, but the file content is cleared and the size becomes zero bytes. This means you can continue to refer to the file with the filename, but the actual content of the file has been deleted
